traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -290,6 +290,57 @@ func GetDSNameAndCDNFromID(tx *sql.Tx, id int) (tc.DeliveryServiceName, tc.CDNNa
 	return dsName, cdnName, true, nil
 }
 
+// GetFederationResolversByFederationID fetches all of the federation resolvers currently assigned to a federation.
```

Comment:
True, but it returns three things and this only describes one of them.

Reply:
It also sort of seems like since this returns a slice there's no real point to the second return value, which is `false` if and only if an error occurred. Typically that's used to indicate whether or not the requested object exists, but in this case if there are no rows you'd just return an empty slice.